server/filestore.go

```diff
@@ -64,6 +64,12 @@ type FileConsumerInfo struct {
 	ConsumerConfig
 }
 
+// Default file and directory permissions.
+const (
+	defaultDirPerms  = os.FileMode(0750)
```

Comment:
Good to make those a constant so we can change in one place. But if we are going to do a 2.2.5, did we not say for 2.2.4 that we were not comfortable changing from 755 to 750 for an update release?

Reply:
At last minute for 2.2.4 yes but been thinking through it for awhile now and I think its ok. /cc @philpennock